[35mDEBUG   [0m src.steps.common:common.py:18 Running fixture setup: start_1_node
[32mINFO    [0m src.node.status_node:status_node.py:38 Starting node with command: ['./status-cli', 'serve', '-n', 'first_node', '-p', '57442']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - 2024-09-17T06:09:52.906Z	INFO	status-cli/serve.go:34	Running serve command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -name first_node
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -port 57442
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -light false
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -api-modules waku,wakuext,wakuv2,permissions,eth
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -telemetry-server-url
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -fleet status.staging
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - 
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - 2024-09-17T06:09:52.906Z	INFO	first_node	status-cli/util.go:56	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - 2024-09-17T06:09:55.572Z	INFO	first_node	status-cli/util.go:69	account created, key UID: 0x579020b127dc267dbb3d6744ac793bf34f34b783c6a30709d73d1f1023f2c2c1
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - 2024-09-17T06:09:55.601Z	INFO	first_node	status-cli/util.go:94	messenger started, public key: 0x043ad753bc2f63cac537d712c338f9cc9772afab768ae67aa1a82cacd8ba205360ea2c49c0f2d482b994c3a4592c1d9bdf19339a013f41cfe3be8b34a981e2d73c
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - 2024-09-17T06:10:00.601Z	INFO	first_node	status-cli/serve.go:100	retrieve messages...
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 10 seconds
[35mDEBUG   [0m tests.conftest:conftest.py:42 Running fixture teardown: save_node_logs_on_fail
[35mDEBUG   [0m tests.conftest:conftest.py:45 Test failed, attempting to rename and move logs to the test_results folder
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file first_node.log to test_results/testleavecommunity_test_leave_community_baseline_first_node.log
[35mDEBUG   [0m tests.conftest:conftest.py:62 Running fixture teardown: clear_open_nodes
[32mINFO    [0m src.node.status_node:status_node.py:72 Stopping node with name: first_node
[35mDEBUG   [0m src.steps.common:common.py:18 Running fixture setup: start_1_node
[32mINFO    [0m src.node.status_node:status_node.py:38 Starting node with command: ['./status-cli', 'serve', '-n', 'first_node', '-p', '52719']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - 2024-09-17T06:10:10.741Z	INFO	status-cli/serve.go:34	Running serve command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -name first_node
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -port 52719
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -light false
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -api-modules waku,wakuext,wakuv2,permissions,eth
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -telemetry-server-url
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -fleet status.staging
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - 
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - 2024-09-17T06:10:10.741Z	INFO	first_node	status-cli/util.go:56	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - 2024-09-17T06:10:13.419Z	INFO	first_node	status-cli/util.go:69	account created, key UID: 0x94f41c2b2728719844862878fab8c914ec806405617f93839502533c6e4a28cd
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - 2024-09-17T06:10:13.443Z	INFO	first_node	status-cli/util.go:94	messenger started, public key: 0x045f4f8b8c9c5f7c5d0255a5c4a2b39b89ded4bd2f19f409933fadfcdc40d172d58db0554f2db88e92c4e0170cf657459a2b9672cf332674005989a595d035faeb
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - 2024-09-17T06:10:18.443Z	INFO	first_node	status-cli/serve.go:100	retrieve messages...
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 10 seconds
[35mDEBUG   [0m tests.conftest:conftest.py:42 Running fixture teardown: save_node_logs_on_fail
[35mDEBUG   [0m tests.conftest:conftest.py:45 Test failed, attempting to rename and move logs to the test_results folder
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file first_node.log to test_results/testleavecommunity_test_leave_community_baseline_first_node.log
[35mDEBUG   [0m tests.conftest:conftest.py:62 Running fixture teardown: clear_open_nodes
[32mINFO    [0m src.node.status_node:status_node.py:72 Stopping node with name: first_node
[35mDEBUG   [0m src.steps.common:common.py:18 Running fixture setup: start_1_node
[32mINFO    [0m src.node.status_node:status_node.py:38 Starting node with command: ['./status-cli', 'serve', '-n', 'first_node', '-p', '51841']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - 2024-09-17T06:10:28.579Z	INFO	status-cli/serve.go:34	Running serve command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -name first_node
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -port 51841
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -light false
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -api-modules waku,wakuext,wakuv2,permissions,eth
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -telemetry-server-url
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -fleet status.staging
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - 
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - 2024-09-17T06:10:28.579Z	INFO	first_node	status-cli/util.go:56	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - 2024-09-17T06:10:31.252Z	INFO	first_node	status-cli/util.go:69	account created, key UID: 0x7b9401ba6d945cec21bafb6c968d4c1122709be017ca03be9793f7b394155a32
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - 2024-09-17T06:10:31.281Z	INFO	first_node	status-cli/util.go:94	messenger started, public key: 0x04f1713ce7452ecfa4eb26f51c482f195e5c2237fbd3262121d0ebaabfe8ff60d6cb26704cc9f666e3274ff6cafc79504f3b80eb4b2c30c2d1fd551dc3c7b8e753
[35mDEBUG   [0m src.node.status_node:status_node.py:64 FIRST_NODE - 2024-09-17T06:10:36.282Z	INFO	first_node	status-cli/serve.go:100	retrieve messages...
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 10 seconds
[35mDEBUG   [0m tests.conftest:conftest.py:42 Running fixture teardown: save_node_logs_on_fail
[35mDEBUG   [0m tests.conftest:conftest.py:45 Test failed, attempting to rename and move logs to the test_results folder
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file first_node.log to test_results/testleavecommunity_test_leave_community_baseline_first_node.log
[35mDEBUG   [0m tests.conftest:conftest.py:62 Running fixture teardown: clear_open_nodes
[32mINFO    [0m src.node.status_node:status_node.py:72 Stopping node with name: first_node